Core/Domain/Recovery/RecoveryState.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Lykke.Service.ClientAccountRecovery.Client.AutoRestClient.Models;
```

Comment:
Is the namespace correct?

Reply:
Yes it seems that namespace is correct. As we discussed to use enums directly from client library to bypass redundant mappings.
![image](https://user-images.githubusercontent.com/12168520/43839018-1362fc70-9b2e-11e8-99bf-a4933049539a.png)